src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -14,8 +14,8 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Session\Session;
-use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\SecurityContext;
```

Comment:
this one is used in the phpdoc of a deprecated function, so it should still be kept until 3.0.